Sources/Misc/Box.swift

```diff
@@ -31,3 +31,5 @@ final class WeakBox<T: AnyObject> {
     init(_ value: T) { self.value = value }
 
 }
+
+extension Box: Sendable where T: Sendable {}
```

Comment:
@aboedo this is all that was needed :)

Reply:
Hi @NachoSoto, do you recall _why_ this did the trick? We're seeing a very similar crash in the firebase-ios-sdk and I'm thinking this solution will also work for our "Box" type.

Reply:
You mean the `Sendable` part, or `Box` in general?

Reply:
The `Sendable` part

Reply:
That wasn't fixing a crash, that was necessary to use this type inside another Sendable type.

Reply:
Oh, my bad. I re-read the issue and see that now. Sorry for the pings.

Reply:
No problem!